lms/static/js/spec/edxnotes/views/tabs/tags_spec.js

```diff
@@ -0,0 +1,80 @@
+define([
+    'jquery', 'underscore', 'js/common_helpers/template_helpers', 'js/spec/edxnotes/helpers',
+    'js/edxnotes/collections/notes', 'js/edxnotes/collections/tabs',
+    'js/edxnotes/views/tabs/tags', 'js/spec/edxnotes/custom_matchers',
+    'jasmine-jquery'
+], function(
+    $, _, TemplateHelpers, Helpers, NotesCollection, TabsCollection, TagsView,
+    customMatchers
+) {
+    'use strict';
+    describe('EdxNotes TagsView', function() {
```

Comment:
I chose to just do bok choy tests of scrollToTag functionality, as we have been asked to reduce redundant tests, and verifying the scrolling behavior in a unit tests seems very difficult/impossible. I did cover all 4 views (most recent, course structure, search, tags) in the bok choy tests.